lock.go

```diff
@@ -53,7 +53,7 @@ func readLock(r io.Reader) (*lock, error) {
 	for i, ld := range rl.P {
 		r := gps.Revision(ld.Revision)
 
-		var v gps.Version
+		var v gps.Version = r
```

Comment:
> Rewriting it like the style you mentioned earlier (a := defaultVal) should fix it.

The warning is wrong - at least for me, that doesn't compile. `gps.Version` is an interface and `r` is a struct.

> There's one more on status.go:97

It's a doc error, and I was lazy. Hence "happier", not "happy" :)